### ISSUE TITLE
Webpack error when launching cypress component-test using material-ui Popover or Modal

### ISSUE THREAD

**Issue by francois-robert** [nrwl/nx#11655]
## Current Behavior

Webpack is launching an error (see bellow) when launching cypress component-test on a components using material-ui Popover or Modal. It seems the issue is with @popperjs/core.

## Expected Behavior

Webpack should launch without errors.

## Steps to Reproduce

- Create a react app using nx
- Create a component using material-ui Popover or Modal
- Add Cypress component-test to react app
- Create test on component
- Launch component-test

Example repository : https://github.com/francois-robert/component-test-popperjs-error

### Failure Logs

```
ERROR in ../../node_modules/@popperjs/core/lib/index.js
Module build failed (from ../../node_modules/babel-loader/lib/index.js):
Error: Cannot find module './.config/babel.config'
Require stack:
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\files\configuration.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\files\index.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\index.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\babel-loader\lib\index.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\loader-runner\lib\loadLoader.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\loader-runner\lib\LoaderRunner.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\webpack\lib\NormalModuleFactory.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\webpack\lib\Compiler.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\webpack\lib\webpack.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\webpack\lib\index.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@nrwl\web\src\utils\config.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@nrwl\web\src\utils\web.config.js
- C:\Users\franc\Documents\Code\behindthescreen\node_modules\@nrwl\react\plugins\component-testing\index.js
- C:\Users\franc\Documents\Code\behindthescreen\apps\client-web\cypress.config.ts
- C:\Users\franc\AppData\Local\Cypress\Cache\10.3.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\run_require_async_child.js
- C:\Users\franc\AppData\Local\Cypress\Cache\10.3.1\Cypress\resources\app\node_modules\@packages\server\lib\plugins\child\require_async_child.js
    at Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Module._resolveFilename (C:\Users\franc\AppData\Local\Cypress\Cache\10.3.1\Cypress\resources\app\node_modules\tsconfig-paths\lib\register.js:75:40)
    at Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue (C:\Users\franc\AppData\Local\Cypress\Cache\10.3.1\Cypress\resources\app\node_modules\@cspotcode\source-map-support\source-map-support.js:679:30)
    at Function.module_1.default._resolveFilename (C:\Users\franc\AppData\Local\Cypress\Cache\10.3.1\Cypress\resources\app\node_modules\@packages\server\node_modules\@cypress\webpack-dev-server\dist\helpers\sourceRelativeWebpackModules.js:107:16)
    at resolve (internal/modules/cjs/helpers.js:99:19)
    at loadConfig (C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\files\configuration.js:172:5)
    at loadConfig.next (<anonymous>)
    at mergeExtendsChain (C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\config-chain.js:387:45)
    at mergeExtendsChain.next (<anonymous>)
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\config-chain.js:373:20
    at Generator.next (<anonymous>)
    at loadFileChain (C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\config-chain.js:232:24)
    at loadFileChain.next (<anonymous>)
    at buildRootChain (C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\config-chain.js:135:31)
    at buildRootChain.next (<anonymous>)
    at loadPrivatePartialConfig (C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\partial.js:103:62)
    at loadPrivatePartialConfig.next (<anonymous>)
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\config\partial.js:149:25
    at Generator.next (<anonymous>)
    at step (C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:261:32)
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:273:13
    at async.call.result.err.err (C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:223:11)
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:189:28
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\@babel\core\lib\gensync-utils\async.js:74:7
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:113:33
    at step (C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:287:14)
    at C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:273:13
    at async.call.result.err.err (C:\Users\franc\Documents\Code\behindthescreen\node_modules\gensync\index.js:223:11)

webpack 5.65.0 compiled with 1 error in 19708 ms
```

### Environment

This is nx report for my project, not for the example github repo.

```
$ nx report

 >  NX   Report complete - copy this into the issue template

   Node : 14.18.0
   OS   : win32 x64
   npm  : 6.14.15

   nx : 14.5.4
   @nrwl/angular : Not Found
   @nrwl/cypress : 14.5.4
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.5.4
   @nrwl/eslint-plugin-nx : 14.5.4
   @nrwl/express : 14.5.4
   @nrwl/jest : 14.5.4
   @nrwl/js : 14.5.4
   @nrwl/linter : 14.5.4
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : 14.5.4
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 14.5.4
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.5.4
   @nrwl/web : 14.5.4
   @nrwl/workspace : 14.5.4
   typescript : 4.4.4
   ---------------------------------------
   Community plugins:
```

**Comment by francois-robert** [comment]
Hello ! Do you need more elements to reproduce ?

**Comment by barbados-clemens** [comment]
Thanks for reporting and providing a reproduce-able. I've confirmed the most recent changes in #11725 fix this and will be released soonish.

**Comment by francois-robert** [comment]
Thanks ! I've just tried to migrate to NX 14.6.2 but this warning appears 
![image](https://user-images.githubusercontent.com/5152483/187957197-ad55efa2-ead0-46c6-9336-4b1be1653aab.png)

Therefore the same error occurs. Shoul I change build configuration in order to use the new webpack build feature ?

**Comment by barbados-clemens** [comment]
@francois-robert when migrating did nx not update the 'component-test' target to include a devServerTarget property?

**Comment by francois-robert** [comment]
Il did, but it does not seem to use it :
![image](https://user-images.githubusercontent.com/5152483/188081974-f724fbdb-c163-4cec-a297-aba96bfffc3b.png)

**Comment by barbados-clemens** [comment]
what does your cypress.config.ts file look like?

**Comment by francois-robert** [comment]
It seems ok : 
```import { defineConfig } from 'cypress';
import { nxComponentTestingPreset } from '@nrwl/react/plugins/component-testing';

export default defineConfig({
  component: nxComponentTestingPreset(__filename),
});
```

**Comment by barbados-clemens** [comment]
@francois-robert can you push up the changes to the repo you shared so I can take a look?
https://github.com/francois-robert/component-test-popperjs-error

**Comment by francois-robert** [comment]
Hello @barbados-clemens 
Sorry I took a few days off.

I've juste updated the repo (https://github.com/francois-robert/component-test-popperjs-error) with 14.7.4 and had the same error. The changes have been pushed.

**Comment by francois-robert** [comment]
Issue is solved with lastest version (14.7.6).

Thanks @barbados-clemens
